pkg/networkmanager/ip-settings.jsx

```diff
@@ -130,7 +130,25 @@ export const IpSettingsDialog = ({ topic, connection, dev, settings }) => {
 
         return false;
     };
-
+    const addressToMask = (address) => {
+        const addSplited = address.split('.');
```

Comment:
"splited" is not a word (and "splitted" isn't either, English is weird). I suggest to just rename it to "split".

Reply:
ack